automatic/mkvtoolnix/mkvtoolnix.nuspec

```diff
@@ -24,10 +24,11 @@ With these tools one can get information about (mkvinfo) Matroska files, extract
     <projectSourceUrl>https://mkvtoolnix.download/source.html</projectSourceUrl>
     <docsUrl>https://mkvtoolnix.download/docs.html</docsUrl>
     <dependencies>
-      <dependency id="chocolatey-core.extension" version="1.0" />
+      <dependency id="chocolatey-core.extension" version="1.3.3" />
```

Comment:
missing dependency on `chocolatey` version `0.10.5` or higher due to the use of `file64`